Tests/QueueInteropTransportTest.php

```diff
@@ -428,6 +429,47 @@ public function testThrow()
         $transport->send($envelope);
     }
 
+    /**
+     * @expectedException \Symfony\Component\Messenger\Exception\MessageDecodingFailedException
```

Comment:
Same here

Reply:
@Nyholm updated into expectException + added the missing reject call from the Consumer.